config/config.exs

```diff
@@ -4,3 +4,7 @@ config :phoenix, :json_library, Jason
 config :phoenix, :trim_on_html_eex_engine, false
 config :logger, :level, :debug
 config :logger, :backends, []
+
+config :phoenix, :template_engines, [
+  heex: Phoenix.LiveView.HTMLEngine
+]
```

Comment:
We shouldn't need this config. Instead, change mix.exs to use phoenix from github on the v1.5 branch which already has this. We can ship a new v1.5 soon.